packages/core/src/base/body.scss

```diff
@@ -44,7 +44,7 @@ Style guide: style.base
       &:active,
       &:focus,
       &:hover {
-        color: $color-base-inverse;
+        color: $color-muted-inverse;
```

Comment:
I just did the calculation, the exact color of white at 80% opacity on the inverse dark bg is `#cfd5dc`. I wonder if we should use that instead, considering we use `$color-muted-inverse` as the disabled transparent button styling.